docs/src/main/sphinx/connector/atop.rst

```diff
@@ -0,0 +1,147 @@
+==============
+Atop connector
+==============
+
+The Atop connector supports reading disk utilization statistics from the ``atop``
+(Advanced System and Process Monitor) Linux server performance analysis tool.
+
+Requirements
+------------
+
+In order to use this connector, the machine on which the Trino worker is running
+needs to have the ``atop`` tool installed locally.
+
+The system utilization statistics are internally read from
+``/var/log/atop/atop_YYYYMMDD`` files which correspond to the each day in the last
+``atop.max-history-days`` days. These files contain compressed binary data written
+on a ``10`` minutes interval by ``/etc/atop/atop.daily`` cron daemon.
+
+Connector configuration
+-----------------------
+
+Create ``etc/catalog/system_monitor.properties``
+
+.. code-block:: text
+
+    connector.name=atop
+    atop.executable-path=/usr/bin/atop
+
+Configuration
+-----------------------
+
+.. list-table:: Configuration variables
+  :widths: 40, 10, 10, 40
+  :header-rows: 1
+
+  * - Property name
+    - Default value
+    - Required
+    - Description
+  * - ``atop.concurrent-readers-per-node``
+    - ``1``
+    - Yes
+    - The number of concurrent read operations allowed per node.
+  * - ``atop.executable-path``
+    - (none)
+    - Yes
+    - The file path on the local file system for the ``atop`` utility
+  * - ``atop.executable-read-timeout``
+    - ``1ms``
+    - Yes
+    - The timeout when reading from the atop process.
```

Comment:
Do we read from the process? Or from files created by the process?

Reply:
We read from the process (which subsequently reads from the files previously and stored within `/var/log/atop/atop_YYYYMMDD` files